Front-End/public/connect-esp-intro.css

```diff
@@ -0,0 +1,78 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+main {
+    font-family: "Poppins", sans-serif;
+!important;
+    font-weight: 400;
+}
+
+body {
+    font-family: Arial, sans-serif;
+    margin: 0;
+    padding: 0;
+    color: #333;
+    background-color: #f9f9f9;
+}
+
+main {
+    max-width: 800px;
+    margin: 20px auto;
+    background: #fff;
+    padding: 20px;
+    border: 1px solid #ddd;
```

Comment:
Unnecessary. Please remove this line. The border can't be seen anyway.

Reply:
This was done so the box can look better. There's no need to remove it.

Reply:
I see, disregard it.